### PR TITLE
Update 11.2 + Update 16

### PR DESCRIPTION
**Pull request by Rainb0wKey**

### CODE REVIEW

File: src/guides/awaitedCommands.md [hunk 1]
```diff
@@ -2,67 +2,148 @@
...
-To create an awaited command, following function and callbacks are used -
-
-### $awaitFunc[]
-`$awaitFunc` is a function. It's used to initiate an awaited command.
-
-#### Usage
+## Content
+[**Used**](#used) > [**Supported Filters**](#supported-filters) > [**$awaitFunc[]**](#awaitfunc) > [**$awaitedCommand[]**](#awaitedcommand) > [**$awaitedCommandError[]**](#awaitedcommanderror) > [**Create Await command**](#create-await)
+
+## Used
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md]: If you could change this header in `buttons.md` too?
```suggestion
## Functions Used
```

File: src/guides/awaitedCommands.md [hunk 2]
```diff
@@ -2,67 +2,148 @@
...
+- [`$awaitFunc[]`](../bdscript/awaitFunc.md)
+- [`$awaitedCommand[]`](../callbacks/awaitedCommand.md)
+- [`$awaitedCommandError[]`](../callbacks/awaitedCommandError.md)
+## Supported Filters
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md:13]:
```suggestion

## Supported Filters
```

File: src/guides/awaitedCommands.md [hunk 3]
```diff
@@ -2,67 +2,148 @@
...
+- `<numeric>`: Accepts only number input.
+- `<word1/word2/...>`: Accepts only specified words provided inside `<>`. Use `/` as a separator for multiple words.
+
+# $awaitFunc
+Used to initiate an awaited command.
+## Syntax
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md:20]:
```suggestion

## Syntax
```

File: src/guides/awaitedCommands.md [hunk 4]
```diff
@@ -2,67 +2,148 @@
...
+```
+$awaitFunc[name;(user ID;channel ID)]
 ```
-$awaitFunc[command name;(user ID;channel ID)]
+### Parameters
+- `command name` `(Type: String || Flag: Required)`: The name used inside [`$awaitedCommand[]`](#awaitedcommand) and [`$awaitedCommandError[]`](#awaitedcommanderror) callbacks.
+- `user ID` `(Type: Snowflake || Flag: Vacantable)`: The user the awaited command will trigger for. Uses command author, if `user ID` is not given.
+- `channel ID` `(Type: Snowflake || Flag: Optional)`: The channel where the command will be awaited. Uses current channel, if `channel ID` is not given.
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md:28]:
```suggestion

### Parameters
- `command name` `(Type: String || Flag: Required)`: The name used inside [`$awaitedCommand[]`](#awaitedcommand) and [`$awaitedCommandError[]`](#awaitedcommanderror) callbacks.
- `user ID` `(Type: Snowflake || Flag: Vacantable)`: The user the awaited command will trigger for. Uses command author, if `user ID` is not given.
- `channel ID` `(Type: Snowflake || Flag: Optional)`: The channel where the command will be awaited. Uses current channel, if `channel ID` is not given.

```

File: src/guides/awaitedCommands.md [hunk 5]
```diff
@@ -2,67 +2,148 @@
...
+$nomention
+What do you want me to say?
+$awaitFunc[say]
+```
+![example](https://user-images.githubusercontent.com/113303649/212294420-acf01905-c9f5-4673-99f0-375f9d786f25.png)
 
-### $awaitedCommand[]
-`$awaitedCommand` is a callback. It gets triggered when an awaited command gets initiated.
+# $awaitedCommand
+_Triggered when an awaited command gets initiated._
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md]: This callback only runs after `$awaitFunc[]` was run and the given user sent a message in the given channel aka responded. So calling this initiation is not correct.
```suggestion

# $awaitedCommand
_Triggered when an awaited command gets responded to._
```

File: src/guides/awaitedCommands.md [hunk 6]
```diff
@@ -2,67 +2,148 @@
...
 
-#### Usage
+`$awaitedCommand[]` is a callback, which means it's used in the command trigger (not the code). The command is ran when an awaited command gets initiated.
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md]:
```suggestion
`$awaitedCommand[]` is a callback, which means it's used in the command trigger (not the code). The command is only run when an awaited command gets responded to.

```

File: src/guides/awaitedCommands.md [hunk 7]
```diff
@@ -2,67 +2,148 @@
...
 $awaitedCommand[name;(filter)]
 ```
-- `name` - The name which is used in [`$awaitFunc`](#awaitfunc) function.
-- `filter` - It is used to limit user input. Supported filters are 
-    - `<numeric>` - Accepts only number input.
-    - `<word1/word2>` - Accepts only those specified words provided inside `<>`. Use `/` as separator for multiple words.
-    - `empty` - Accepts any input from the user. To use this filter, keep `filter` argument empty.
+### Parameters
+- `name`: The name used in [`$awaitFunc[]`](#awaitfunc) function.
+- `filter`: It is used to limit user input. [Supported Filters](#supported-filters)
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md]:
```suggestion

### Parameters
- `name`: The name used in [`$awaitFunc[]`](#awaitfunc) function.
- `filter`: Used to limit user input. [Supported Filters](#supported-filters)

```

File: src/guides/awaitedCommands.md [hunk 8]
```diff
@@ -2,67 +2,148 @@
...
+You have provided a number: $message
+```
+![example](https://user-images.githubusercontent.com/113303649/212089433-e998259a-0e74-4401-9140-a7ea4c6c3776.png)
+
+# $awaitedCommandError
+_Triggered when an awaited command doesn't match with provided filter._
 
-### $awaitedCommandError[]
-`$awaitedCommandError[]` is a callback. It gets triggered when an awaited command doesn't match with provided filter.
+`$awaitedCommandError[]` is a callback, which means it's used in the command trigger (not the code). The command is ran when an awaited command doesn't match with provided filter.
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md]:
```suggestion
`$awaitedCommandError[]` is a callback, which means it's used in the command trigger (not the code). The command is only run when an awaited command doesn't match with provided filter.
```

File: src/guides/awaitedCommands.md [hunk 9]
```diff
@@ -2,67 +2,148 @@
...
+$nomention
+Invalid number!
+```
+![example](https://user-images.githubusercontent.com/113303649/212291974-4efa0531-d896-4aa3-ac1a-b8fa7d6794d0.png)
 
-   ![Screenshot_20220813_072224](https://user-images.githubusercontent.com/95774950/184464158-083019b0-821a-4683-a969-02293b44f86a.png)
-   ![Screenshot_20220813_072236](https://user-images.githubusercontent.com/95774950/184464161-62689486-928a-4aa1-900f-8ea295fc9437.png)
-   ![Screenshot_20220813_072322](https://user-images.githubusercontent.com/95774950/184464163-50eac8f9-6194-4763-ac20-e18a2c97d47d.png)
-   ![Screenshot_20220813_072131](https://user-images.githubusercontent.com/95774950/184464150-79857481-578f-4f4b-b725-126f35a88ad4.png)
+# Create Await
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md]:
```suggestion
# Creating an awaited command
```

File: src/guides/awaitedCommands.md [hunk 10]
```diff
@@ -2,67 +2,148 @@
...
-
-## Getting Started
-To create an awaited command, following function and callbacks are used -
-
-### $awaitFunc[]
-`$awaitFunc` is a function. It's used to initiate an awaited command.
-
-#### Usage
+## Content
+[**Used**](#used) > [**Supported Filters**](#supported-filters) > [**$awaitFunc[]**](#awaitfunc) > [**$awaitedCommand[]**](#awaitedcommand) > [**$awaitedCommandError[]**](#awaitedcommanderror) > [**Create Await command**](#create-await)
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md]:
```suggestion
[**Used**](#used) > [**Supported Filters**](#supported-filters) > [**$awaitFunc[]**](#awaitfunc) > [**$awaitedCommand[]**](#awaitedcommand) > [**$awaitedCommandError[]**](#awaitedcommanderror) > [**Creating an awaited command**](#creating-an-awaited-command)
```

File: src/guides/awaitedCommands.md [hunk 11]
```diff
@@ -2,67 +2,148 @@
...
+![example](https://user-images.githubusercontent.com/113303649/212291974-4efa0531-d896-4aa3-ac1a-b8fa7d6794d0.png)
 
-   ![Screenshot_20220813_072224](https://user-images.githubusercontent.com/95774950/184464158-083019b0-821a-4683-a969-02293b44f86a.png)
-   ![Screenshot_20220813_072236](https://user-images.githubusercontent.com/95774950/184464161-62689486-928a-4aa1-900f-8ea295fc9437.png)
-   ![Screenshot_20220813_072322](https://user-images.githubusercontent.com/95774950/184464163-50eac8f9-6194-4763-ac20-e18a2c97d47d.png)
-   ![Screenshot_20220813_072131](https://user-images.githubusercontent.com/95774950/184464150-79857481-578f-4f4b-b725-126f35a88ad4.png)
+# Create Await
+## Without filter
+1. Create two commands with `!say` and `$awaitedCommand[say;]` triggers.
+2. Paste the following code: Code for the command with the `!say` trigger:
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md]:
```suggestion

1. Create two commands with `!say` and `$awaitedCommand[say;]` triggers.
2. Paste the following code:

Code for the `!say` command:
```

File: src/guides/awaitedCommands.md [hunk 12]
```diff
@@ -2,67 +2,148 @@
...
+```
+$nomention
+What do you want me to say?
+$awaitFunc[say]
+```
+Code for the command with the `$awaitedCommand[say;]` trigger:
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md:116]:
```suggestion

Code for the command with the `$awaitedCommand[say;]` trigger:
```

File: src/guides/awaitedCommands.md [hunk 13]
```diff
@@ -2,67 +2,148 @@
...
+```
+$nomention
+$message
+```
+3. Execute command `!say`
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md:122]:
```suggestion

3. Execute command `!say`
```

File: src/guides/awaitedCommands.md [hunk 14]
```diff
@@ -2,67 +2,148 @@
...
 
-   > ⚠️ The above example requires **BDScript 2** since it contains `$elseif`.
+![example](https://user-images.githubusercontent.com/113303649/212294420-acf01905-c9f5-4673-99f0-375f9d786f25.png)
+## With choose filter
+1. Create two commands with `!odd` and `$awaitedCommand[odd;<yes/no/cancel>]` triggers.
+2. Paste the following code: Code for the command with the `!odd` trigger:
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md]:
```suggestion

1. Create two commands with `!odd` and `$awaitedCommand[odd;<yes/no/cancel>]` triggers.
2. Paste the following code:

Code for the `!odd` command:
```

File: src/guides/awaitedCommands.md [hunk 15]
```diff
@@ -2,67 +2,148 @@
...
+```
+$nomention
+Is '19' an odd number?
+$awaitFunc[odd]
+```
+Code for the command with the `$awaitedCommand[say;]` trigger:
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md]:
```suggestion

Code for the command with the `$awaitedCommand[say;]` trigger:
```

File: src/guides/awaitedCommands.md [hunk 16]
```diff
@@ -2,67 +2,148 @@
...
+$nomention
+$if[$message==yes]
+   Your answer is correct!
+$elseif[$message==no]
+   Your answer is incorrect!
+$elseif[$message==cancel]
+   Command cancelled!
+$endif
+```
+3. Execute command `!odd`
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md:149]:
```suggestion

3. Execute command `!odd`
```

File: src/guides/awaitedCommands.md [hunk 17]
```diff
@@ -2,67 +2,148 @@
...
 
-- Numeric Filter
+![example](https://user-images.githubusercontent.com/113303649/212088333-54a94584-f854-45cf-8b7e-6980aa370764.png)
+## With numeric filter
+1. Create two commands with `!number` and `$awaitedCommand[number;<numeric>]` triggers.
+2. Paste the following code: Code for the command with the `!number` trigger:
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md]:
```suggestion

1. Create two commands with `!number` and `$awaitedCommand[number;<numeric>]` triggers.
2. Paste the following code:

Code for the `!number` command:
```

File: src/guides/awaitedCommands.md [hunk 18]
```diff
@@ -2,67 +2,148 @@
...
+```
+$nomention
+Provide a number!
+$awaitFunc[number]
+```
+Code for the command with the `$awaitedCommand[number;<numeric>]` trigger:
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md:164]:
```suggestion

Code for the command with the `$awaitedCommand[number;<numeric>]` trigger:
```

File: src/guides/awaitedCommands.md [hunk 19]
```diff
@@ -2,67 +2,148 @@
...
+```
+$nomention
+You have provided a number: $message
+```
+3. Execute command `!number`
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md:170]:
```suggestion

3. Execute command `!number`
```

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

**Review by MineBartekSA — CHANGES_REQUESTED** [review]
Change the `## Used` header to `## Functions Used` in `buttons.md`

**Comment by Rainb0wKey** [comment]
ok

File: src/guides/awaitedCommands.md [hunk 20]
```diff
@@ -2,67 +2,171 @@
 
 Awaited commands are a special type of command where the bot waits for the user's response.
 
-![end result](https://i.imgur.com/rRRcIXA.jpg) 
+## Content
+[**Used**](#used) > [**Supported Filters**](#supported-filters) > [**$awaitFunc[]**](#awaitfunc) > [**$awaitedCommand[]**](#awaitedcommand) > [**$awaitedCommandError[]**](#awaitedcommanderror) > [**Creating an awaited command**](#creating-an-awaited-command)
```
Comment (MineBartekSA) [inline src/guides/awaitedCommands.md]:
```suggestion
[**Functions Used**](#functions-used) > [**Supported Filters**](#supported-filters) > [**$awaitFunc[]**](#awaitfunc) > [**$awaitedCommand[]**](#awaitedcommand) > [**$awaitedCommandError[]**](#awaitedcommanderror) > [**Creating an awaited command**](#creating-an-awaited-command)
```

**Review by MineBartekSA — CHANGES_REQUESTED** [review]
I forgot about this

**Review by MineBartekSA — APPROVED** [review]
Thank you for your hard work

**Comment by MineBartekSA** [comment]
@Rainb0wKey please handle merge conflict

**Comment by Rainb0wKey** [comment]
ok

**Review by MineBartekSA — APPROVED** [review]

**Pull request merged**